scripts/minikube.sh

```diff
@@ -95,6 +95,9 @@ up)
 
     enable_psp
 
+    # /run/systemd/resolve/resolv.conf is required
```

Comment:
I have not seen any issues in atleast in my local deployment, everything works even without this.

Reply:
Really? And you have the /run/systemd/resolve/resolv.conf file on the host where you run the containers? Was this on a cleanly installed system, or were there possibly some Kubernetes containers still running? Some distributions provide the syetemd-resolved service by default, others do not.

The `EXTRA_CONFIG` points to `--extra-config=kubelet.resolv-conf=/run/systemd/resolve/resolv.conf` and without that file, kubelet should not be able to resolve hostnames to download further container images or similar.

Reply:
Alternatively we could check if the file exists, and only then append the parameter to `EXTRA_CONFIG`. Is that something you would prefer?

Reply:
Note this is the vm created from minikube where the specify the vmdriver, currently i use kvm2.

Reply:
> Alternatively we could check if the file exists, and only then append the parameter to `EXTRA_CONFIG`. Is that something you would prefer?

yes that would be good check this only the extra config is present

Reply:
how you will come to know that this file is missing inside a VM? currently, this is checked only on the host machine if the VM driver is none

Reply:
We hit the problem with `--vm-driver=none`, so in that case it is easy to verify. If the default VM that minikube starts has the file, it should cover the common scenarios. (You can pass `--iso-url=centos`, I think, to install a CentOS VM. But we do not use that in our CI.)

Reply:
okay, in that case, we will run this check if the VM driver is none and extra config is provided isn't it?

Reply:
Updated this PR with a different approach. Please review again @Madhu-1 @Yuggupta27